packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -648,6 +648,35 @@ describe('<TreeItem />', () => {
           expect(getByTestId('five')).to.have.attribute('aria-expanded', 'true');
           expect(getByTestId('six')).to.have.attribute('aria-expanded', 'false');
         });
+
+        it('should not expand siblings that are end nodes', () => {
+          const { getByTestId } = render(
+            <TreeView>
+              <TreeItem nodeId="one" label="one" data-testid="one">
+                <TreeItem nodeId="two" label="two" data-testid="two" />
+              </TreeItem>
+              <TreeItem nodeId="three" label="three" data-testid="three">
+                <TreeItem nodeId="four" label="four" data-testid="four" />
+              </TreeItem>
+              <TreeItem nodeId="five" label="five" data-testid="five">
+                <TreeItem nodeId="six" label="six" data-testid="six">
+                  <TreeItem nodeId="seven" label="seven" data-testid="seven" />
+                </TreeItem>
+              </TreeItem>
+              <TreeItem nodeId="eight" label="eight" data-testid="eight" />
+            </TreeView>,
+          );
+
+          getByTestId('one').focus();
+          expect(getByTestId('one')).not.to.have.class('Mui-expanded');
```

Comment:
Is this how we check for the expanded state in the other tests as well? I think we can improve this test by asserting `aria-expanded`. I guess leaf nodes should not have this attribute at all while the other should have it set to true | false?

Reply:
Ok I understand now that this test doesn't concern end nodes. I think we can put the assertions in the `expands all siblings that are at the same level as the current node` test. Seem like that one is doing the same but is only concerned with `aria-expanded`. It's a bit concerning that elements could be `aria-expanded="false"` but `class="Mui-expanded".` 

/cc @joshwooding

Reply:
Ok